backend/mapservice/Controllers/EdpController.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System;
+using System.IO;
+using System.Text;
+using System.Web;
+using System.Web.Mvc;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using MapService.Models;
+using log4net;
+using MapService.Components;
+using System.Configuration;
+using Newtonsoft.Json.Linq;
+using System.Net;
+using System.Xml;
+using EdpConn;
+
+/// <summary>
+/// Denna Controller sköter kommunikation med EDP Vision mha klassen ImplEdpConnectorPublic som finns i projektet EdpConn.
+/// Vill man kompilera en mapservice utan referenser till EDP följer man instruktionerna i filen EdpConn.cs i projektet EdpConn.
+///
+/// För att slå på kommunkationen måste man lägga till följande rader till Web.config i sektion <appSettings>
+/// <add key = "edpUUID" value="421623AA-D1D3-42EE-A516-28DC2EEFA544" />
+/// <add key = "edpClientName" value="KommunGIS" />
+/// <add key = "edpServerUrl" value="http://censrv364:64235/" />
+/// <add key = "edpRemoveDomainFromUserName" value="1" />
+/// </summary>
```

Comment:
**Perhaps we should discuss** if this should be done the other way around: if an admin wants to __include__ support for some external system (EDP in this case), the admin must uncomment stuff to make it work. In other words: external plugins disabled by default, line 15 commented out.